docs/installation.md

```diff
@@ -7,7 +7,7 @@ This tool can be natively installed on your machine or can be containerized and
 
 ## Prerequisites
 
-- Any machine with Python 3.6+ installed.
+- Any machine with Python 3.7+ installed.
```

Comment:
Maybe add a note on which version dropped 3.6 so anyone using it can peg to that if they need to

Reply:
It was part of the build dependencies. I'm not sure we ever supported 3.6, this might have been a typo.